### PR TITLE
Enabling Policheck

### PR DESCRIPTION
**Pull request by harshit7962**
Enabling PoliCheck and TSA configs with 1ES Templates.
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/wpf/pull/9722)

### CODE REVIEW

**Comment by harshit7962** [comment]
@mmitche these changes are in accordance to our discussions [here](https://github.com/dotnet/wpf/pull/9424).

Can we go ahead and merge this? Will this be sufficient for enabling Policheck runs?

File: eng/tsaoptions.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "instanceUrl": "https://devdiv.visualstudio.com/",
+    "template": "TFSDEVDIV",
+    "projectName": "DEVDIV",
+    "areaPath": "DevDiv\\NET Fundamentals\\NET Partners\\WPF\\Security\\SDL",
+    "iterationPath": "DevDiv",
+    "notificationAliases": [ "wpf-netcore-dev@microsoft.com" ],
```
Comment (singhashish-wpf) [inline eng/tsaoptions.json:7]: This exists?

**Review by singhashish-wpf — APPROVED** [review]

File: eng/tsaoptions.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "instanceUrl": "https://devdiv.visualstudio.com/",
+    "template": "TFSDEVDIV",
+    "projectName": "DEVDIV",
+    "areaPath": "DevDiv\\NET Fundamentals\\NET Partners\\WPF\\Security\\SDL",
```
[the remark this answers is not in the document]
Comment (harshit7962) [reply eng/tsaoptions.json:5]: Yes this area path exists.

File: eng/tsaoptions.json (hunk 1 above)
Comment (harshit7962) [reply eng/tsaoptions.json:7]: The alias is not in use now, updated it to a correct one.

**Review by singhashish-wpf — APPROVED** [review]

**Pull request merged**